docker.Makefile

```diff
@@ -14,10 +14,11 @@ endif
 BUILD_ARGS = \
   --build-arg BUILD_BASE=${BUILD_BASE_IMAGE} \
   --build-arg RUN_BASE=${RUN_BASE_IMAGE} \
-  --build-arg BUILDTIME=${BUILDTIME} \
-  --build-arg GITCOMMIT=${GITCOMMIT} \
-  --build-arg VERSION=${VERSION} \
-  --build-arg IMAGE_REPO_PREFIX=${IMAGE_REPO_PREFIX}
+  --build-arg BUILDTIME \
+  --build-arg GITCOMMIT \
+  --build-arg VERSION \
+  --build-arg IMAGE_REPO_PREFIX \
+  --build-arg=GOPROXY
```

Comment:
I didn't get the reason of this changes.
Variables BUILDTIME, GITCOMMIT, VERSION and IMAGE_REPO_PREFIX lost their values.
GOPROXY with an '=' sign and no value... 

That looks strange to me

Reply:
--build-arg GITCOMMIT by default just get value from the matching environment variable, so is equivalent to --build-arg GITCOMMIT=${GITCOMMIT}